ios/FluentUI/People Picker/PersonaListView.swift

```diff
@@ -92,6 +92,22 @@ open class PersonaListView: UITableView {
 
         dataSource = self
         delegate = self
+
+        NotificationCenter.default.addObserver(self,
+                                               selector: #selector(themeDidChange),
+                                               name: .didChangeTheme,
+                                               object: nil)
+    }
+
+    @objc private func themeDidChange(_ notification: Notification) {
+        guard let window = window, window.isEqual(notification.object) else {
+            return
+        }
+        updateBackgroundColor()
```

Comment:
Should this not also have updateColors?

Reply:
This is the only color update in `PersonaListView`